ESLint linebreak-style giving errors

## Bug description

*Please describe.*  
While editing files on vscode, it gives error at every line stating

```bash
   92:23  error  Expected linebreaks to be 'LF' but found 'CRLF'               linebreak-style
  92:23  error  Delete `␍`                                                    prettier/prettier
```

Changing the rule from

```JSON
		'linebreak-style': ['error', 'unix'],
```
to

```JSON
		'linebreak-style': ['error', 'windows'],
```

gives another error as

```bash
92:23  error  Delete `␍`                                                    prettier/prettier
```
## Expected behavior

The editor should show/ not show errors based on the Operating System being used by the user.

## How to reproduce

1. Clone this repository and switch to `main` branch.
2. Open the cloned repository in vscode.
3. You will see errors and squiggly lines at ending of every line.

## Version information 
* **Signoz version**:

       - Version independent

* **Browser version**:

        - Browser version independent.

* **Your OS and version**:

      - Windows 10
       - 21H1

## Additional context

This can fixed either by using `process.platform` to check whether the OS is windows or not. If yes, then we can change the rule to be applied according to `windows` and if not windows, then switch it to `unix`. I tried this way but I still got the error even after I disabled the rule it gives out `92:23  error  Delete `␍`                                                    prettier/prettier`. This also needs to be reproduced to check if it does the same for everyone.

#### *Thank you* for your bug report – we love squashing them!

--- a/frontend/.eslintrc.js
+++ b/frontend/.eslintrc.js
@@ -42,7 +42,7 @@ module.exports = {
 		'@typescript-eslint/explicit-function-return-type': 'error',
 		'@typescript-eslint/no-var-requires': 0,
 		'react/no-array-index-key': 2,
-		'linebreak-style': ['error', 'unix'],
+		'linebreak-style': ['error', process.platform == 'win32' ? 'windows' : 'unix'],
 
 		// simple sort error
 		'simple-import-sort/imports': 'error',

--- a/frontend/.eslintrc.js
+++ b/frontend/.eslintrc.js
@@ -42,7 +42,7 @@ module.exports = {
 		'@typescript-eslint/explicit-function-return-type': 'error',
 		'@typescript-eslint/no-var-requires': 0,
 		'react/no-array-index-key': 2,
-		'linebreak-style': ['error', process.platform == 'win32' ? 'windows' : 'unix'],
+		'linebreak-style': ['error', process.platform === 'win32' ? 'windows' : 'unix'],
 
 		// simple sort error
 		'simple-import-sort/imports': 'error',

